app/javascript/components/RequestCart.vue

```diff
@@ -16,7 +16,6 @@
             variation="text"
             class="denied-button"
             aria-labelledby="closeCart"
-            focused
```

Comment:
Unless there's a good reason to focus this field, I think it's best to apply the focus as it was, at the top of the panel. This is because it may not be obvious to a user with a screen reader that the focus is in the middle of the panel.

Reply:
I had to get rid of this because with it around it kept re-rendering the template on unfocusing the note field, causing it to wipe out whatever was typed.

Reply:
@tpendragon, I see. One solution might be to store the note in the vuex store on keydown to prevent that. I will approve this as is and attempt to solve that focus issue in another ticket.